src/sync/branch.test.ts

```diff
@@ -119,7 +119,7 @@ describe("sync/branches", () => {
 				});
 		};
 
-		const nockBranchRequest = (response, variables?: Record<string, any>) =>
+		const nockBranchRequest = (response: object, variables?: Record<string, unknown>) =>
```

Comment:
`unknown` is better than `any` as the ts prevents code from using any fields from that object.